middlewares/auth.middleware.js

```diff
@@ -0,0 +1,91 @@
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
+
+const models = require('../models');
+const { commonErrorHandler } = require('../helpers/common-function.helper');
+
+const checkAccessToken = async (req, res, next) => {
+  try {
+    const header = req.headers['authorization'];
+    const accessToken = header ? header.split(' ')[1] : null;
+    if (!accessToken) {
+      throw new Error('Access denied');
+    }
+    const decodedJwt = jwt.verify(
+      accessToken,
+      process.env.SECRET_KEY_ACCESS_TOKEN,
+    );
+
+    const user = await models.User.findOne({
+      where: {
+        id: decodedJwt.userId,
+      },
+    });
+    if (!user) {
+      throw new Error('User Not found');
+    }
+    req.user = user;
+    next();
+  } catch (error) {
+    res.statusCode = 401;
+    if (res.message === 'User Not found') res.statusCode = 404;
+    commonErrorHandler(req, res, error.message, res.statusCode, error);
+  }
+};
+
+const checkRefreshToken = async (req, res, next) => {
+  try {
+    const { refreshToken } = req;
+    const decodedRefreshToken = jwt.verify(
+      refreshToken,
+      process.env.SECRET_KEY_REFRESH_TOKEN,
```

Comment:
have you included these env variables in .env.sample file?

Reply:
Yes